thunder/core/proxies.py

```diff
@@ -1041,13 +1047,8 @@ def _infer_tensor_properties(
         thunder_fsdp_padding_size if thunder_fsdp_padding_size is not None else _thunder_fsdp_padding_size
     )
 
-    # Extracts actual values for shape
-    # TODO RC1 Enable this
-    if using_symbolic_values():
-        raise NotImplementedError(
-            f"Trying to construct a tensor proxy while using symbolic values, but this is not yet supported"
-        )
-
+    # dynamic shape not yet enabled, otherwise, the bake in should be guarded.
```

Comment:
Can we add a condition that will break when we add dynamic shapes, so we don't forget to guard this when it's time?